src/TreeTenantCounter/App.svelte

```diff
@@ -75,7 +100,8 @@
             </div>
           {/if}
         </div>
-
+        {/if}
+        {#if goal && goal != "" && goal != 0}
```

Comment:
Same as above. Is `{#if goal}` not enough to check?

Reply:
@sarveshpro and I tried to solve that with minimal code but only this satisfied every condition

Reply:
Well, I couldn't reproduce errors using `{#if goal}`, but better be safe than sorry :-)